magda-web-client/src/Components/Dataset/Add/Pages/People/OrgUnitDropdown.tsx

```diff
@@ -38,29 +41,31 @@ export default function OrgUnitDropdown({
             </div>
         );
     } else {
-        const onChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
-            const value = event.target.value;
-            onChangeCallback(value);
-        };
+        const value = result.find(option => option.id === orgUnitId);
 
         return (
-            <select
-                className="au-select"
-                defaultValue={orgUnitId || ""}
-                onChange={onChange}
-            >
-                <option value="" disabled>
-                    Select a team
-                </option>
-
-                {result.map(val => {
-                    return (
-                        <option key={val.id} value={val.id}>
-                            {val.name}
-                        </option>
-                    );
-                })}
-            </select>
+            <Select
+                className="react-select"
+                isMulti={false}
+                isSearchable={false}
+                onChange={(rawValue, action) => {
+                    const value = rawValue as (
+                        | { value: string }
+                        | undefined
+                        | null);
+                    console.log(value);
```

Comment:
the debug code to be removed

Reply:
@AlexGilleran It seems this `console.log` (in `Components/Dataset/Add/Pages/People/OrgUnitDropdown.tsx`) still there